flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/stream/GroupAggregateTestPrograms.java

```diff
@@ -383,11 +399,10 @@ public class GroupAggregateTestPrograms {
                                             "avg_a DOUBLE",
                                             "min_c VARCHAR",
                                             "PRIMARY KEY (b) NOT ENFORCED")
-                                    .consumedBeforeRestore(
+                                    .consumedValues(
```

Comment:
What's the point here?`consumedValues` invokes `consumedBeforeRestore`.

Reply:
I think even though they internally use the same method, my intention in using `consumedValues` is to indicate that this test is not concerned with the data before and after the restore.

Reply:
My concern is this test spec does not test the `state_ttl` hint. Even if we remove the hint, the result remains unchanged.

Reply:
I have updated the state ttl hint to "1s" to let state in savepoint always has expired at first.